Package.swift

```diff
@@ -40,6 +40,8 @@ let package = Package(
         .package(url: "https://github.com/scinfu/SwiftSoup", from: "2.7.0"),
         .package(url: "https://github.com/aus-der-Technik/FileMonitor", from: "1.2.0"),
         .package(url: "https://github.com/Zollerboy1/SwiftCommand", from: "1.4.0"),
+        .package(url: "https://github.com/johnfairh/swift-sass", from: "3.1.0")
```

Comment:
My bad, dart sass should be commented out, it's for an upcoming sass / css feature.